### PR TITLE
Add an endpoint to get the list of validators with evm compatible addresses

### LINKED ISSUE #586: Add an endpoint to get the list of validators with EVM compatible addresses
The endpoint will work very similarly to `/staking/validators` with the added field that shows the addresses for EVM bridges.

### PR DESCRIPTION
**Pull request by prin-r**
Fix #586

### CODE REVIEW

File: chain/client/client.go [hunk 1]
```diff
@@ -230,6 +230,7 @@ func ServeSwaggerUI() http.Handler {
 func RegisterRoutes(cliCtx context.CLIContext, r *mux.Router) {
 	r.HandleFunc("/d3n/blocks/latest", LatestBlocksRequestHandlerFn(cliCtx)).Methods("GET")
 	r.HandleFunc("/d3n/txs/latest", LatestTxsRequestHandlerFn(cliCtx)).Methods("GET")
+	r.HandleFunc("/staking/validators", GetValidators(cliCtx)).Methods("GET")
```
Comment (sorawit) [inline chain/client/client.go]: Won't this collide with the existing `/staking/validators`? Let's call this `/bandchain/evm-validators`

File: chain/client/validators.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/types/rest"
+)
+
+type ValidatorMinimal struct {
+	Address     string `json:"address"`
+	VotingPower int64  `json:"voting_power"`
+}
+
+type ValidatorsOnEthereum struct {
```
Comment (sorawit) [inline chain/client/validators.go]: Don't think this has anything specific with Ethereum.

File: chain/client/validators.go [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+	BlockHeight int64              `json:"block_height"`
+	Validators  []ValidatorMinimal `json:"validators"`
+}
+
+func GetValidators(cliCtx context.CLIContext) http.HandlerFunc {
```
Comment (sorawit) [inline chain/client/validators.go]: Let's call this `GetEVMValidators`

**Review by sorawit — CHANGES_REQUESTED** [review]

File: chain/client/validators.go [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+			}
+
+			if pubkey, err := crypto.DecompressPubkey(pubKeyBytes[:]); err != nil {
+				rest.WriteErrorResponse(w, http.StatusInternalServerError, err.Error())
+				return
+			} else {
+				validatorsOnETH.Validators = append(
+					validatorsOnETH.Validators,
+					ValidatorMinimal{
+						Address:     fmt.Sprintf("0x%x", crypto.PubkeyToAddress(*pubkey)),
```
Comment (taobun) [inline chain/client/validators.go]: Can we use `crypto.PubkeyToAddress(*pubkey).String()` instead?

**Review by taobun — CHANGES_REQUESTED** [review]

File: chain/client/validators.go (hunk 4 above)
Comment (prin-r) [reply chain/client/validators.go]: done

File: chain/client/validators.go [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+			return
+		}
+
+		validators, err := node.Validators(nil)
+		if err != nil {
+			rest.WriteErrorResponse(w, http.StatusInternalServerError, err.Error())
+			return
+		}
+
+		validatorsOnETH := ValidatorsMinimal{}
```
Comment (taobun) [inline chain/client/validators.go]: EVM or ETH just choose one

**Review by taobun — CHANGES_REQUESTED** [review]

File: chain/client/validators.go (hunk 5 above)
Comment (prin-r) [reply chain/client/validators.go]: done

**Review by sorawit — APPROVED** [review]

**Review by taobun — APPROVED** [review]

**Pull request merged**